src/main/java/org/jabref/gui/groups/GroupTree.css

```diff
@@ -30,13 +30,12 @@
 }
 
 .numberColumn > .hits:any-selected {
-    -fx-background-color: derive(-jr-green, 70%);
+    -fx-background-color: derive(-jr-gray-3, 70%);
```

Comment:
With these changes, the number column is now also displayed in gray in the light theme. This is probably also not that readable.

I would propose to add two colors (`-jr-groups-any-hits` and `-jr-groups-all-hits`), which are used here. Then the dark theme can overwrite them.